plugins/events-backend-module-bitbucket-server/package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "@backstage/plugin-events-backend-module-bitbucket-server",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.0",
```

New packages should start at `0.0.0`